jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -595,7 +599,7 @@ YUI.add('juju-env-api', function(Y) {
       var cback = function(err, response) {
         this.handleLogin({error: err, response: response});
         if (callback) {
-          callback(err);
+          callback(err, response);
```

Comment:
I think this change is not required, and does not reflect the docstring of this method.